airbyte-integrations/connectors/destination-elasticsearch/src/main/java/io/airbyte/integrations/destination/elasticsearch/ElasticsearchConnection.java

```diff
@@ -109,9 +110,25 @@ protected Header[] configureHeaders(ConnectorConfiguration config) {
         headerList.add(new BasicHeader("Authorization", basicHeader));
       }
     }
+
+    headerList.add(new BasicHeader("user-agent", this.getConnectorVersion()));
+
     return headerList.toArray(new Header[headerList.size()]);
   }
 
+  /**
+   * Read the version of the connector from the environment
+   * @return the version of the connector
+   */
+  private String getConnectorVersion() {
```

Comment:
Looks like that would return the name of the connector alongside the version, i.e. `airbyte-destination-elasticsearch@0.1.7`, right? Should the function be called something like `makeConnectorUAString`, if it's also only used for the user agent?

Reply:
Yeah. This makes sense to me. I will rename the function. 

Thanks 👍🏻